### ISSUE TITLE
Introduction doesn't show business avatar on load

### ISSUE THREAD

**Issue by morenoh149** [smooch/smooch-js#393]
Hello.
I'm using this library. Loading the min file from the cdn. I created a react component component that calls `Smooch.init()` and `Smooch.render()` inside `componentDidMount`.
The widget loads but the default avatar is shown. Once I send a message, the correct avatar gets displayed. Is there some conflict with how this library was designed to be used and how I'm trying to use it?

**Comment by lemieux** [comment]
Hey @morenoh149, what do you mean by the default avatar?

**Comment by morenoh149** [comment]
the black building with white outlines.

**Comment by lemieux** [comment]
Hmmm, I'm not sure it has anything to do with React, but I'll try to repro that.

**Comment by lemieux** [comment]
In which browser are you having this behavior and which version of the lib are you using? I just tried the latest in Chrome in a React app and I can't seem to be able to repro the problem.

**Comment by morenoh149** [comment]
chrome v52. And the smooch lib from the cdn. https://cdn.smooch.io/smooch.min.js

**Comment by lemieux** [comment]
Yeah, sorry I can't repro the problem. I just tried the latest version through npm and through the CDN and it doesn't happen. Can you provide a link to it or sample code for me to reproduce the problem?

I'll close this issue for now until I have more information because I can't reproduce the problem.

**Comment by lemieux** [comment]
We're  you using the embedded mode?

**Comment by morenoh149** [comment]
not sure to be perfectly honest. We solved the glitch by hiding the avatar and app name via css.

```css
.sk-intro-section .app-icon {
  display: none;
}

.sk-intro-section .app-name {
  display: none;
}

#smooch-container #sk-container.sk-embedded #sk-conversation .sk-intro-section .intro-text {
  margin-left: 0;
}

#smooch-container #sk-container.sk-embedded #sk-conversation .sk-intro-section .app-name {
  margin-left: 0;
}
```

**Comment by lemieux** [comment]
Okay, I'm asking because we have someone in the team who is having a similar problem to yours when it's embedded. I'll make sure to reopen this issue when/if I can confirm they are the same problem.

I'll keep you posted!

**Comment by lemieux** [comment]
I'm reopening this issue because the problem is confirmed and will be fixed by #417.
